### ISSUE TITLE
Lvgl bugs the display

### ISSUE THREAD

**Issue by Trackhe** [Bodmer/TFT_eSPI#878]
The Display Shows the JPEG for the Intro and then is Freeze. Maybe it have a problem with the missing touch?
What would you appreciate?

![image](https://user-images.githubusercontent.com/1494328/102924634-42b4ef00-4492-11eb-8515-3af3ea037e3f.jpeg)

**Comment by Bodmer** [comment]
Do the TFT_eSPI examples work with the display?

**Comment by Trackhe** [comment]
Yes

**Comment by Bodmer** [comment]
OK, that suggests the problem is in your sketch.

**Comment by Trackhe** [comment]
i try the spi display without touch in code. i will see if this happend when i use the spi display with the exakt same code

**Comment by Trackhe** [comment]
The Display with SPI works with the exact same sketch.

**Comment by Bodmer** [comment]
Can you post a copy of the sketch so I can test it and track the problem down?

**Comment by Trackhe** [comment]
I try to reproduce it with an example sketch. in the next days.

**Comment by Bodmer** [comment]
I loaded lvgl and configured it. I can confirm that the two examples (ESP32_TFT_eSPI and ESP32_TFT_eSPI_Slider) work with a SPI display.

The ESP32_TFT_eSPI example runs on and ESP32 with 8 bit parallel interface but the ESP32_TFT_eSPI_Slider example will not. I tried disabling the touch functions in that non-working sketch but could not get it to run at all. The same hardware setup does run the TFT_eSPI examples OK. So it appears there may be a conflict between the libraries for some reason in 8 bit parallel mode.  I do not see this as a TFT_eSPI library issue and thus will not be investigating further. In the case where someone identifes the need for a simple change then this will be examined for incorporation.
